clang-tools-extra/test/clang-tidy/checkers/bugprone/implicit-widening-of-multiplication-result-pointer-offset.cpp

```diff
@@ -18,7 +18,7 @@ char *t0(char *base, int a, int b) {
   // CHECK-NOTES-CXX:                  static_cast<ptrdiff_t>( )
   // CHECK-NOTES-ALL: :[[@LINE-5]]:17: note: perform multiplication in a wider type
   // CHECK-NOTES-C:                    (ptrdiff_t)
-  // CHECK-NOTES-CXX:                  static_cast<ptrdiff_t>()
+  // CHECK-NOTES-CXX:                  static_cast<ptrdiff_t>( )
```

Comment:
NOTE: This is legacy issue and shoudn't be fixed under this PR, but all those checks with CHECK-NOTES looks invalid. We should use CHECK-MESSAGES and CHECK-FIXES, in that case we could verify that fixed line after applying fix looks correctly instead of checking clang-tidy output.

Reply:
I agree with you, like I've mentionned :

> Since both suggestions are overlapping clang-tidy refuses to apply them [...]

This is probably the reason why they are currently not using `CHECK-FIXES`. It would be a good idea to update the tests to use `CHECK-FIXES` but we would probably need to remove one of the FIX-IT for it to work.